src/universal/styles/ui.js

```diff
@@ -360,7 +355,7 @@ const ui = {
   menuBackgroundColor: '#fff',
   menuBorderColor: appTheme.palette.mid30l,
   menuBorderRadius: borderRadiusSmall,
-  menuBoxShadow: shadow[2],
+  menuBoxShadow: shadow[3],
```

Comment:
i hear ya. another option is for the shadow to be dynamic & depth to be passed down from a parent.
For example, we calculate the card depth based on hover/focus. Then, we pass in depth + 1 to the menu. this only feels weird when someone tabs away from the card & it goes from 2 to 0, which means the menu would go from 3 to 1. to soften that blow, i added super subtle box-shadow transitions in my branch in case you'd like to incorporate those into this PR :wink: